src/main/java/com/hms_networks/americas/sc/extensions/mqtt/ConstrainedMqttManager.java

```diff
@@ -665,6 +674,23 @@ public void setQos(int mqttQos) throws Exception {
     }
   }
 
+  /**
+   * Sets the time interval (in secs) for sending keep alive messages on the MQTT client connection.
+   * If the MQTT manager is currently running, the MQTT connection will be updated with the new keep
```

Comment:
Do you know if this should be set prior to start() being called? I think this comment might be misleading.  My guess is that when this option is set after the manager is already running there is no change i.e., this parameter is read when a connection is established.

Reply:
That is a good point. The Ewon MQTT implementation does not have developer documentation, and we only have a forum post to go off of: https://techforum.ewon.biz/thread-586.html

I traced the call down through the ETK MqttClient class, and it goes all the way into a native method, which we do not have visibility in to. 

It is probably best to assume that the MqttClient will not dynamically update the keep alive interval.

I will update this PR as follows:
1. Add (optional) constructor parameter for specifying keep alive interval.
2. Make `setMqttKeepAliveIntervalSecs` trigger an MQTT restart similar to the other parameters.